docs/extensions/leveraging-ui-components.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Learn how to use the Directus UI components in your extension projects.
+contributors: Esther Agbaje
+---
+
+# Leveraging Directus UI Components
+
+Directus provides a set of reusable UI components that can be used in when building extensions. These components are
+designed to provide a consistent user experience across different projects and extensions.
+
+## Components Playground
+
+Explore the available components on the [Histoire Components Playground](https://components.directus.io/). Here, you can
+interact with different components and experiment with variations before implementing them in your project.
+
+![Directus UI Components](https://marketing.directus.app/assets/46d72f1a-5d9f-49f8-a09a-b2ffc5200812.gif)
+
+## Usage within Extensions
+
+Directus UI components are globally registered, making them accessible throughout your extension project without the
+need for explicit imports. Here’s a basic example:
+
+```tsx
+<VButton>
+  My Button
+</VButton>
+```
```

Comment:
```suggestion
Directus UI components are globally registered, making them accessible throughout your extension project without the
need for explicit imports. Here’s an example:
```

Reply:
A reminder: https://justsimply.dev/

Reply:
Was this supposed to remove the example?